lib/internal/Magento/Framework/Webapi/Test/Unit/InputLimit/DefaultPageSizeSetterTest.php

```diff
@@ -48,7 +48,7 @@ public function testPageSizeIsNotSetWhenLimitingIsDisabled()
             ->getMock();
         $searchCriteria->method('getPageSize')
             ->willReturn(null);
-        $searchCriteria->expects(self::never())
+        $searchCriteria->expects(self::any())
```

Comment:
Hello @quterorta,

Instead of using `any` here, can we use `atLeastOnce`. The reason for the same is by using any() we are actually not sure if the method was called at all, such a check becomes useless.